python/openai/openai_frontend/engine/utils/triton.py

```diff
@@ -77,6 +81,8 @@ def _create_trtllm_inference_request(
         inputs["random_seed"] = np.uint64([[request.seed]])
     if request.temperature is not None:
         inputs["temperature"] = np.float32([[request.temperature]])
+    # NOTE: TRT-LLM doesn't currently support runtime changes of 'echo' and it
+    # is configured at model load time, so we don't handle it here for now.
```

Comment:
Can we check at load time and error if a request doesn't match the load time parameter?

Reply:
This one is a bit tedious because `exclude_input_in_output` is a config field of the `tensorrt_llm_bls` model, but the model we'd be interacting with at this point is the BLS or Ensemble model. Would need some global-er assumption about a single TRT-LLM model that we store somewhere, or a way to associate with TRT-LLM model we're interacting with through the BLS/Ensemble model.

Will think about it a bit more tomorrow - but would rather exclude that change from this PR. I'll also look for an update on the ticket for TRT-LLM adding runtime support for this flag to see when it may be added.